### PR TITLE
[7.5.x] Adding gating.yaml file for CVP to be persisted in dist-git repo

### PR DESCRIPTION
**Pull request by jakubschwan**
Signed-off-by: Jakub Schwan <jschwan@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by goldmann** [comment]
This change itself will not make these files available in the dist-git repositories. You are probably looking for this: http://docs.cekit.io/en/latest/descriptor/image.html#osbs-extra-directory

**Comment by jakubschwan** [comment]
@goldmann thanks didn't realize that. Can you please take a second look if it is look ok now?

File: businesscentral-monitoring/image.yaml [hunk 1]
```diff
@@ -156,6 +156,7 @@ packages:
 osbs:
       configuration:
           container_file: container.yaml
+      extra_dir: custom-files
```
Comment (errantepiphany) [inline businesscentral-monitoring/image.yaml]: Can we call this something more specific then "custom-files"? That is vague and doesn't really say what the purpose is. I think even the name "osbs_extra" (from the cekit docs) would be fine, as the name implies what it's for.

Comment (jakubschwan) [reply businesscentral-monitoring/image.yaml]: dir can called "cvp" as the content is required for cvp, but in future there can be more files that are not related to cvp so "osbs-extra" will be probably better option

Comment (errantepiphany) [reply businesscentral-monitoring/image.yaml]: Agreed that it's a better idea to make it a name for any extra files that osbs might need - whether it be now or in the future.

Comment (jakubschwan) [reply businesscentral-monitoring/image.yaml]: @errantepiphany dir name updated

Comment (errantepiphany) [reply businesscentral-monitoring/image.yaml]: thanks; looks good

**Pull request merged**